src/imitation/algorithms/dagger.py

```diff
@@ -508,7 +508,7 @@ def create_trajectory_collector(self) -> InteractiveTrajectoryCollector:
         beta = self.beta_schedule(self.round_num)
         collector = InteractiveTrajectoryCollector(
             venv=self.venv,
-            get_robot_acts=lambda obs: self.bc_trainer.policy.predict(obs)[0],
+            get_robot_acts=lambda acts: self.bc_trainer.policy.predict(acts)[0],
```

Comment:
`predict` takes an **obs**ervation and returns **act**ion**s** (so I think code was right before).

Reply:
Yes I agree. Actually, the `acts` is from master branch. I planned to change `acts` to `obs` and in the end put it under #802 . You see `acts` here because I revert that change and put it under #802. So I would prefer still keep it here and fix it in #802.